scripts/ffmprovisr

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+if [[ $OSTYPE = darwin* ]] ; then
+    if [ -d /usr/local/Cellar/ffmprovisr ] ; then
+        ffmprovisr_path=$(find /usr/local/Cellar/ffmprovisr -iname 'index.html' | sort -M | tail -n1)
+    fi
+    if [ -z "${ffmprovisr_path}" ] ; then
+        ffmprovisr_path='https://amiaopensource.github.io/ffmprovisr/'
+    fi
+    open "${ffmprovisr_path}"
+fi
+
+if [[ $OSTYPE = linux-gnu ]] ; then
```

Comment:
I certainly could change it to use `elif`. With anticipating more conditionals for different systems (like linux on windows) I personally find the symmetry of having individual discrete `if fi` blocks easier to read, but I might be in the minority on that!